contracts/reputation/AbstractProposalManager.sol

```diff
@@ -18,7 +18,7 @@ import "../traits/ChargesEthFee.sol";
 
 contract AbstractProposalManager is Initializable, ChargesEthFee {
 
-  uint256 constant VERSION = 2;
+  uint256 public constant VERSION = 2;
```

Comment:
this increases contract size in KB, I remember there were problems with it

Reply:
but it should be public to check it from frontend